content/Rustup-1.28.2.md

```diff
@@ -0,0 +1,85 @@
++++
+path = "2025/04/25/Rustup-1.28.2"
+title = "Announcing rustup 1.28.2"
+authors = ["The Rustup Team"]
+aliases = ["2025/04/25/Rustup-1.28.2.html"]
++++
+
+The rustup team is happy to announce the release of rustup version 1.28.2.
+[Rustup][install] is the recommended tool to install [Rust][rust], a programming language that is
+empowering everyone to build reliable and efficient software.
+
+## What's new in rustup 1.28.2
+
+The headlines of this release are:
+
+- The cURL download backend is now officially deprecated and a warning will start to show up
+  when it is used. [pr#4277]
+
+  - While rustup predates reqwest and rustls, the rustup team has long wanted to standardize on
```

Comment:
```suggestion
  - The rustup team has long wanted to standardize on
```

Reply:
This seems like useful context to explain why we didn't start out with reqwest/rustls?

Reply:
Maybe, but for a notification post like this, I'm not so worried about why the *previous* decision was made but more interested in what's changing, what I need to do about it, and then maybe why the change is happening. If historical context is needed, I'd hope it could come after the other points have been addressed.

Reply:
I agree with @djc with this, so I guess it'd still be better to keep it here 🙏